src/kvilib/ext/KviCryptEngine.cpp

```diff
@@ -128,25 +128,25 @@
 		[big]The protocol[/big][br]
 		Well, there is no protocol actually, only the existing implementations, that
 		can be accessed by anyone that want to reproduce them. There are only some
-		points relating to the crypting engines that need to be cleared:[br]
-		The crypted text must be suitable to be sent through an IRC connection;
-		this means that some characters can not appear in the crypted text (e.g. CR,LF,NULL....).
-		KVIrc solves it in a simple way: the crypted binary data is encoded,
+		points relating to the encrypting engines that need to be cleared:[br]
+		The encrypted text must be suitable to be sent through an IRC connection;
+		this means that some characters can not appear in the encrypted text (e.g. CR,LF,NULL....).
+		KVIrc solves it in a simple way: the encrypted binary data is encoded,
 		either as hexadecimal numeric string or in base64.[br]
```

Comment:
Yep they all need proof reading properly, Im not sure I have enough years on my life left.

Thanks.